include/benchmark/benchmark_api.h

```diff
@@ -455,6 +455,16 @@ class Benchmark {
       _benchmark_, n, __LINE__) BENCHMARK_UNUSED =           \
       (new ::benchmark::internal::Benchmark(#n "<" #a "," #b ">", n<a, b>))
 
+#if __cplusplus >= 201103L
+#define BENCHMARK_TEMPLATE(n, ...) \
```

Comment:
can you add documentation to the comment above (and/or the README) regarding the use of this?